src/js/components/DataTable/index.d.ts

```diff
@@ -117,6 +121,7 @@ export interface DataTableProps<TRowType = any> {
         expand?: (string | number)[];
         onExpand?: (expandedKeys: (string | number)[], datum: TRowType) => void;
       };
+  expandAriaLabel?: ({ row }: { row: TRowType }) => string | string;
```

Comment:
I know I suggested this object approach now, but now I'm thinking maybe we just match the structure of `rowDetails` and pass it back as `(row) => `